packages/terra-layout/tests/wdio/layout-slide-panel-spec.js

```diff
@@ -1,33 +1,32 @@
 /* globals $ */
 Terra.describeViewports('Toggle the slide panel and hidden styles', ['medium'], () => {
   before(() => {
-    browser.url('/#/raw/tests/terra-layout/layout/layout-slide-panel-example');
+    browser.url('/raw/tests/terra-layout/layout/layout-slide-panel-example');
     browser.waitForExist('[class*=_panel_][aria-hidden="true"]');
   });
-
-  Terra.it.matchesScreenshot({ selector: '#site' });
+  it('Toggles the slide panel and hidden styles', () => {
+    Terra.validates.element('default', { selector: '#site' });
+  });
```

Comment:
we can have it() just like below without before() since here we have only one it() 
```suggestion
  it('Toggles the slide panel and hidden styles', () => {
     browser.url('/raw/tests/terra-layout/layout/layout-slide-panel-example');
     browser.waitForExist('[class*=_panel_][aria-hidden="true"]');
     Terra.validates.element('default', { selector: '#site' });
  });
```